.circleci/config.yml

```diff
@@ -974,18 +974,18 @@ commands:
   prepare_ubuntu_percona57:
     steps:
     - run: sudo percona-release setup -y ps57
-    - run: sudo apt-get install -y libperconaserverclient20 percona-server-client-5.7 libperconaserverclient20-dev
+    - run: sudo apt-get install -y gdb libperconaserverclient20 percona-server-client-5.7 libperconaserverclient20-dev
```

Comment:
You should not edit the .circleci/config.yml file as it is generated by .circleci/circleci_config_builder.sh

Reply:
What should be done instead?

Reply:
I changed `circleci_config_builder.sh`